org.hectabit.PageBurger.json

```
@@ -0,0 +1,30 @@
+{
+  "id": "org.hectabit.PageBurger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "pageburger",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc"
+  ],
+  "modules": [
+    {
+      "name": "PageBurger",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://centrifuge.hectabit.org/HectaBit/PageBurger-App.git",
+          "tag": "1.0",
+          "commit": "26fae71110cea6568e5c86b6746dbd1a303bcdb7"
+        }
+      ],
+      build-commands": [
+        "make install DESTDIR=/app/bin/ SHAREDIR=/app/share/"
+      ]
```

```suggestion
      "build-commands": [
        "make install DESTDIR=/app/bin/ SHAREDIR=/app/share/"
      ]
```

This should be before sources